docker/Dockerfile-run

```diff
@@ -14,6 +14,9 @@ RUN mkdir build
 WORKDIR /usr/local/src/valhalla/build
 RUN cmake .. -DCMAKE_BUILD_TYPE=RelWithDebInfo
 RUN make all -j$(nproc)
+RUN make utrecht_tiles
+RUN make -j$(nproc) tests
+RUN make -j$(nproc) check
```

Comment:
i mean i know we can do this, but i wonder what is the point. the run docker build only happens when we merge to `master` and we only ever do that after all the checks have passed. this just feels like something that will take extra time in CI but not benefit us.